pkg/sdn/api/validation/validation.go

```diff
@@ -14,6 +15,10 @@ import (
 func ValidateClusterNetwork(clusterNet *sdnapi.ClusterNetwork) field.ErrorList {
 	allErrs := validation.ValidateObjectMeta(&clusterNet.ObjectMeta, false, oapi.MinimalNameRequirements, field.NewPath("metadata"))
 
+	if clusterNet.Name != sdnapi.ClusterNetworkDefault {
```

Comment:
OpenShift SDN solution only looks at 'default' cluster network. What's the use of allowing arbitrary cluster network?